common-rendering/src/components/LastUpdated.tsx

```diff
@@ -17,8 +17,11 @@ const LastUpdated = ({
 				color: ${neutral[46]};
 			`}
 		>
-			<time dateTime={new Date(lastUpdated).toISOString()}>
-				{`Updated at ${lastUpdatedDisplay}`}
+			<time
+				data-last-updated={lastUpdated.toISOString()}
+				dateTime={lastUpdated.toISOString()}
+			>
+				{`Updated: ${lastUpdatedDisplay}`}
```

Comment:
Looks good to me! I only have one observation / question: 
I was checking the Storybook diffs and since this is a common component, this change affects how DCR liveblocks look but (as expected) these don't have the new last updated time formatting, e.g.: https://www.chromatic.com/test?appId=5dfcbf3012392c0020e7140b&id=61b207a3b26af2003a53193f

1. Are we okay with this?
2. Do we want to introduce the new last updated time format in DCR liveblocks as a separate task later?

Reply:
Ah, great point :) I think we would want to update how this displays on DCR too. Currently it's formatted in Frontend (https://github.com/guardian/frontend/blob/main/common/app/model/dotcomrendering/DotcomRenderingSupportTypes.scala#L84). I'll create a card to cover this requirement before this PR is merged :)

Reply:
NB: I've created a card in the backlog to cover this: https://trello.com/c/4GgN73e9